ui/admin/tests/acceptance/workers/list-test.js

```diff
@@ -68,6 +73,7 @@ module('Acceptance | workers | list', function (hooks) {
 
   test('Users can navigate to workers with only list permission', async function (assert) {
     assert.expect(1);
+    featuresService.enable('byow');
```

Comment:
When a flag is enabled via the service, it is safe to assume that it works since the library should already be unit tested.  The enable call also serves as self-documenting code, clearly indicating to developers that the test relies on the flag.  

There are some cases where we specifically test the negative case:  a flag is _disabled_.  Explicit calls to _disable_ flags are no longer necessary, since they are all disabled by default.  That results in a loss of self-documentation.  However, we can reclaim that lost readability by testing the state of a flag when a test explicitly relies on it being disabled.

Reply:
Upon reflection, a better approach would be to encapsulate feature flagging into test helpers.  In a test a developer might call `withFeatures('byow', 'ssh-target')`.  Internally that would result in two calls:  one to enable the features and one to assert they are enabled (or vice versa).